ringpop-admin.js

```diff
@@ -38,6 +38,8 @@ function main() {
         .command('status', 'Status of members in ring')
         .command('partitions', 'Show partition information of a ring')
         .command('top', 'General membership information')
+        .command('reap', 'Remove the faulty nodes from this nodes membership')
+        .command('heal', 'Start a partition heal coordinated by the target node')
```

Comment:
I'd probably take this one step further and suggest: can we make it so the command doesn't return until the heal is complete?

As an operator I want the command to do what it says on the tin. I care less about the internals of how it actually works.